update examples with schedule mechanism (#5)
boucadair/policy-based-network-acl: draft-ma-opsawg-ucl-acl.md

<CODE ENDS>
~~~~

### Examples

>    Here are some delicated examples of period and recurrence formats defined as

Reviewer: ```suggestion
   Here are some examples to illustrate the use of the period and recurrence formats defined as
```

>    YANG groupings. The data is provided in JSON.

Reviewer: ```suggestion
   YANG groupings. Only the message body is provided with JSON used for encoding {{?RFC7951}}
```

#### Period of Time

>    The period starting at 08:00:00 UTC, on January 1, 2023 and ending at 18:00:00 UTC

Reviewer: ```suggestion
   The example of a period that starts at 08:00:00 UTC, on January 1, 2023 and ends at 18:00:00 UTC
```

>    on December 31, 2025:

Reviewer: ```suggestion
   on December 31, 2025 is encoded as follows:
```

~~~~
{
  "period-of-time": {
    "explicit-start": "2023-01-01T08:00:00Z",
    "explicit-end": "2025-12-01T18:00:00Z"
  }
}
~~~~

>    The period starting at 08:00:00 UTC, on January 1, 2023 and lasting 15 days and

Reviewer: ```suggestion
   An example of a period that starts at 08:00:00 UTC, on January 1, 2023 and lasts 15 days and
```

>    5 hours and 20 minutes:

Reviewer: ```suggestion
   5 hours and 20 minutes is encoded as follows:
```

~~~~
{
  "period-of-time": {
    "start": "2023-01-01T08:00:00Z",
    "duration": "P15DT05:20:00"
  }
}
~~~~

>    The period starting at 08:00:00 UTC, on January 1, 2023 and lasting 20 weeks:

Reviewer: ```suggestion
   Now, consider the example of a period that starts at 08:00:00 UTC, on January 1, 2023 and lasts 20 weeks:
```

~~~~
{
  "period-of-time": {
    "start": "2023-01-01T08:00:00Z",
    "duration": "P20W"
  }
}
~~~~

#### Recurrence Rule

>   Every day in December

Reviewer: ```suggestion
  The following snippet can be used to indicate a daily recurrent in December:
```

~~~
{
  "recurrence": {
    "freq": "daily",
    "bymonth": "12"
  }
}
~~~  

>    10 occurrences that occur every last Saturday of the month:

Reviewer: ```suggestion
   The following snippet can be used to indicate 10 occurrences that occur every last Saturday of the month:
```

Reviewer: Isn't "direction" a leaf-list? Or I'm not reading the latest version of the yang module?

Author: Sorry, the direction is indeed a leaf-list, I will fix the json snippet.

~~~~
{
  "recurrence": {
    "freq": "monthly",
    "count": "10",
    "byday": {
      "direction": "-1",
      "weekday": "saturday"
    }
  }
}
~~~~     

>    The last workday of the month, until December 25, 2023:

Reviewer: ```suggestion
   The following indicates the example of a recurrence that occurs last workday of the month until December 25, 2023:
```

Reviewer: Not sure to get this one. Also, isn't "bysetpos" defined as a leaf-list?

Reviewer 2: Yup, leaf-list and integer - I think. It would limit the number of instances within a specific time period. The text might be improved with "on the"... But just checking the example to make sure it is the case.

"The following indicates the example of a recurrence that occurs on the last workday of the month until December 25, 2023:"

Author: Sorry, you guys are right, I will fix that.
